src/SeleniumLibrary/__init__.py

```diff
@@ -70,6 +71,24 @@ class SeleniumLibrary(DynamicCore):
     - `Shortcuts`
     - `Keywords`
 
+    = Webdrivers =
+
+    In order for SeleniumLibrary to succesfully interact with browser,
+    a webdriver module needs to be present in the host that is executing the
+    tests. Typically this is the host running the robot itself but in case of
```

Comment:
Would it be more accurate say: 
```
host that is running the browser. Typically this is the host running the robot itself but in case of 
Selenium Grid, the webdriver binary must be present in the Selenium Grid node.
```

Reply:
Not really ? You can run robot on host X and  have the SeleniumLibrary loaded as remote library from host Y.  That scenario, webdriver binary needs to be in the Y and has nothing to do with selenium grid.